src/components/download-button/download-button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Button from '@mapbox/mr-ui/button';
+import IconText from '@mapbox/mr-ui/icon-text';
+import PropTypes from 'prop-types';
+
+export default class DownloadButton extends React.PureComponent {
+  render() {
+    const { fileType, href } = this.props;
+    const fileName = href
```

Comment:
Would you mind walking me through what's happening on lines 9-13?

Reply:
@katydecorah this is helpful as it's making me really break this down a bit! I was searching for solutions to pull _just_ the filename without the extension for file types that require `passThroughProps`. I found this Stack Overflow article that was helpful - https://stackoverflow.com/questions/43638163/get-filename-from-url-and-strip-file-extension/43638276. 

This code takes the path and separates it at `/` with split(). It then uses pop() to remove the last element from the array which should be the filename. The replace() function is then used with a regular expression to find the beginning of the extension (`.`) and remove it.

Reply:
_I just realized I can remove `toString` though since `href` is already a string_